docs/how-tos/add-subgraph-schema.md

```diff
@@ -60,49 +60,21 @@ We can now use the `Apollo Rover CLI` to compose the supergraph, as:
 rover supergraph compose --config supergraph-config.yaml --elv2-license accept
 ```
 
+Alternatively, we can use the [`diamondlightsource/graph-federation/workflows/compose`](https://github.com/DiamondLightSource/graph-federation/tree/main/workflows/compose) action in GitHub actions.
```

Comment:
Minor point but it would be good to push this as a standard for supergraph-compliant projects, rather than an alternative

Reply:
Also making a note here to check the link works after merge

Reply:
Good call, will re-structure it to be locally and in CI